### DISCUSSION TITLE
Rollback TSML

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#620] [Q&A] [answered]
It seems as though this plug-in is causing some issues with my server and is causing CPU spikes. I don’t know if anyone else is having the same issues but I would like to roll back to a version before 3.12. Is there a repository somewhere in which I could do that?

**Comment by kiyote33** [comment, marked as answer]
Funny. I answered this question just last week when asked by Nicky in Ontario. If you install WP Rollback from the wordpress plugin directory, it will add a rollback option to the plugins in your installed list.

**Reply by Grateful-Viv** [reply]
There is no repository for previous versions? I fixed this issue, but if in the future, I’d prefer not to use a plugin. Please advise. Thanks.

**Reply by brianw-area24** [reply]
You can download a zip file of previous versions from https://wordpress.org/plugins/12-step-meeting-list/advanced/, scroll to the bottom of this page.
